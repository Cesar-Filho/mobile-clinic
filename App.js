import React from "react";
import { YellowBox } from "react-native";
import { applyMiddleware, createStore } from "redux";
import { Provider } from "react-redux";
import thunk from "redux-thunk";
import promise from "redux-promise";
import reducers from "./src/reducers";
import Home from "./src/Home";

YellowBox.ignoreWarnings(["Remote debugger"]);

const store = applyMiddleware(thunk, promise)(createStore)(reducers);

export default function App() {
  return (
    <Provider store={store}>
      <Home />
    </Provider>
  );
}
