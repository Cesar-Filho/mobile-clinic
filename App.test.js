import React from 'react';
import renderer from 'react-test-renderer';

import App from './App';

describe('<App />', () => {
  it('renderizar sem quebrar', () => {
    const rendered = renderer.create(<App />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
