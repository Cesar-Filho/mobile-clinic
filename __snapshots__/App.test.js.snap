// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renderizar sem quebrar 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RNCSafeAreaView
    onInsetsChange={[Function]}
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      onLayout={[Function]}
      style={
        Object {
          "flex": 1,
          "overflow": "hidden",
        }
      }
    >
      <View
        pointerEvents="box-none"
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <View
          accessibilityElementsHidden={false}
          closing={false}
          gestureVelocityImpact={0.3}
          importantForAccessibility="auto"
          onClose={[Function]}
          onGestureBegin={[Function]}
          onGestureCanceled={[Function]}
          onOpen={[Function]}
          onTransitionStart={[Function]}
          pointerEvents="box-none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
          transitionSpec={
            Object {
              "close": Object {
                "animation": "spring",
                "config": Object {
                  "damping": 500,
                  "mass": 3,
                  "overshootClamping": true,
                  "restDisplacementThreshold": 0.01,
                  "restSpeedThreshold": 0.01,
                  "stiffness": 1000,
                },
              },
              "open": Object {
                "animation": "spring",
                "config": Object {
                  "damping": 500,
                  "mass": 3,
                  "overshootClamping": true,
                  "restDisplacementThreshold": 0.01,
                  "restSpeedThreshold": 0.01,
                  "stiffness": 1000,
                },
              },
            }
          }
        >
          <View
            pointerEvents="box-none"
            style={
              Object {
                "flex": 1,
                "marginTop": 64,
              }
            }
          >
            <View
              collapsable={false}
              onGestureHandlerEvent={[Function]}
              onGestureHandlerStateChange={[Function]}
              style={
                Object {
                  "flex": 1,
                  "transform": Array [
                    Object {
                      "translateX": 0,
                    },
                    Object {
                      "translateX": 0,
                    },
                  ],
                }
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "backgroundColor": "rgb(242, 242, 242)",
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "shadowColor": "#000",
                    "shadowOffset": Object {
                      "height": 1,
                      "width": -1,
                    },
                    "shadowOpacity": 0.3,
                    "shadowRadius": 5,
                    "top": 0,
                    "width": 3,
                  }
                }
              />
              <View
                style={
                  Array [
                    Object {
                      "flex": 1,
                      "overflow": "hidden",
                    },
                    Array [
                      Object {
                        "backgroundColor": "rgb(242, 242, 242)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                <View
                  style={
                    Object {
                      "flex": 1,
                      "flexDirection": "column-reverse",
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                      }
                    }
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "display": "flex",
                            "flexBasis": 0,
                            "flexDirection": "column",
                            "flexGrow": 1,
                            "flexShrink": 1,
                            "paddingBottom": 40,
                            "paddingLeft": 24,
                            "paddingRight": 24,
                            "paddingTop": 0,
                          },
                        ]
                      }
                    >
                      <View
                        onLayout={[Function]}
                        style={
                          Array [
                            Array [
                              Object {
                                "flexBasis": 0,
                                "flexGrow": 2,
                                "flexShrink": 1,
                                "justifyContent": "flex-end",
                              },
                            ],
                            Object {
                              "paddingBottom": 0,
                            },
                          ]
                        }
                      >
                        <Text
                          style={
                            Array [
                              Object {
                                "fontSize": 25,
                                "fontWeight": "bold",
                                "marginBottom": 36,
                              },
                            ]
                          }
                        >
                          FaÃ§a seu login ðŸ”‘
                        </Text>
                        <View
                          style={
                            Array [
                              Object {
                                "marginBottom": 5,
                                "marginLeft": 0,
                                "marginRight": 0,
                                "marginTop": 5,
                              },
                            ]
                          }
                        >
                          <Text
                            style={
                              Array [
                                Object {
                                  "color": "#666666",
                                  "fontSize": 14,
                                  "marginBottom": 5,
                                  "marginLeft": 1,
                                  "marginRight": 0,
                                  "marginTop": 0,
                                },
                              ]
                            }
                          >
                            Nome do usuÃ¡rio
                          </Text>
                          <View
                            isActive={false}
                            style={
                              Array [
                                Object {
                                  "alignItems": "center",
                                  "borderColor": "#abb1b7",
                                  "borderRadius": 3,
                                  "borderStyle": "solid",
                                  "borderWidth": 1,
                                  "display": "flex",
                                  "flexDirection": "row",
                                  "paddingHorizontal": 20,
                                  "paddingVertical": 13,
                                },
                              ]
                            }
                          >
                            <TextInput
                              allowFontScaling={true}
                              autoCompleteType="email"
                              autoCorrect={false}
                              autoFocus={true}
                              blurOnSubmit={false}
                              keyboardType="email-address"
                              onBlur={[Function]}
                              onChangeText={[Function]}
                              onFocus={[Function]}
                              onSubmitEditing={[Function]}
                              rejectResponderTermination={true}
                              returnKeyType="next"
                              style={
                                Array [
                                  Object {
                                    "color": "#333333",
                                    "flexBasis": 0,
                                    "flexGrow": 1,
                                    "flexShrink": 1,
                                    "fontSize": 20,
                                  },
                                ]
                              }
                              underlineColorAndroid="transparent"
                            />
                            <Text />
                          </View>
                        </View>
                        <View
                          style={
                            Array [
                              Object {
                                "marginBottom": 5,
                                "marginLeft": 0,
                                "marginRight": 0,
                                "marginTop": 5,
                              },
                            ]
                          }
                        >
                          <Text
                            style={
                              Array [
                                Object {
                                  "color": "#666666",
                                  "fontSize": 14,
                                  "marginBottom": 5,
                                  "marginLeft": 1,
                                  "marginRight": 0,
                                  "marginTop": 0,
                                },
                              ]
                            }
                          >
                            Senha
                          </Text>
                          <View
                            isActive={false}
                            style={
                              Array [
                                Object {
                                  "alignItems": "center",
                                  "borderColor": "#abb1b7",
                                  "borderRadius": 3,
                                  "borderStyle": "solid",
                                  "borderWidth": 1,
                                  "display": "flex",
                                  "flexDirection": "row",
                                  "paddingHorizontal": 20,
                                  "paddingVertical": 13,
                                },
                              ]
                            }
                          >
                            <TextInput
                              allowFontScaling={true}
                              autoCorrect={false}
                              blurOnSubmit={false}
                              onBlur={[Function]}
                              onChangeText={[Function]}
                              onFocus={[Function]}
                              onSubmitEditing={[Function]}
                              rejectResponderTermination={true}
                              returnKeyType="go"
                              secureTextEntry={true}
                              style={
                                Array [
                                  Object {
                                    "color": "#333333",
                                    "flexBasis": 0,
                                    "flexGrow": 1,
                                    "flexShrink": 1,
                                    "fontSize": 20,
                                  },
                                ]
                              }
                              underlineColorAndroid="transparent"
                            />
                            <Text />
                          </View>
                        </View>
                      </View>
                      <View
                        style={
                          Array [
                            Object {
                              "flexBasis": 0,
                              "flexGrow": 1,
                              "flexShrink": 1,
                              "justifyContent": "flex-end",
                            },
                          ]
                        }
                      >
                        <View
                          accessible={true}
                          focusable={true}
                          isTVSelectable={true}
                          onClick={[Function]}
                          onResponderGrant={[Function]}
                          onResponderMove={[Function]}
                          onResponderRelease={[Function]}
                          onResponderTerminate={[Function]}
                          onResponderTerminationRequest={[Function]}
                          onStartShouldSetResponder={[Function]}
                          style={
                            Object {
                              "backgroundColor": "#733dbe",
                              "borderRadius": 5,
                              "height": 50,
                              "justifyContent": "center",
                              "marginBottom": 0,
                              "marginLeft": 6,
                              "marginRight": 6,
                              "marginTop": 0,
                              "opacity": 1,
                            }
                          }
                        >
                          <Text
                            style={
                              Array [
                                Object {
                                  "color": "#ffffff",
                                  "fontSize": 16,
                                  "textAlign": "center",
                                },
                              ]
                            }
                          >
                            Entrar
                          </Text>
                        </View>
                      </View>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </View>
    </View>
    <View
      pointerEvents="box-none"
      style={
        Object {
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    >
      <View
        accessibilityElementsHidden={false}
        importantForAccessibility="auto"
        onLayout={[Function]}
        pointerEvents="box-none"
        style={null}
      >
        <View
          pointerEvents="none"
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        >
          <View
            style={
              Object {
                "backgroundColor": "rgb(255, 255, 255)",
                "borderBottomColor": "rgb(224, 224, 224)",
                "flex": 1,
                "shadowColor": "rgb(224, 224, 224)",
                "shadowOffset": Object {
                  "height": 0.5,
                  "width": 0,
                },
                "shadowOpacity": 0.85,
                "shadowRadius": 0,
              }
            }
          />
        </View>
        <View
          pointerEvents="box-none"
          style={
            Object {
              "height": 64,
              "maxHeight": undefined,
              "minHeight": undefined,
              "opacity": undefined,
              "transform": undefined,
            }
          }
        >
          <View
            pointerEvents="none"
            style={
              Object {
                "height": 20,
              }
            }
          />
          <View
            pointerEvents="box-none"
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "flexDirection": "row",
                "justifyContent": "center",
              }
            }
          >
            <View
              pointerEvents="box-none"
              style={
                Object {
                  "marginHorizontal": 18,
                  "opacity": 1,
                }
              }
            >
              <Text
                accessibilityRole="header"
                numberOfLines={1}
                onLayout={[Function]}
                style={
                  Object {
                    "color": "rgb(28, 28, 30)",
                    "fontSize": 17,
                    "fontWeight": "600",
                  }
                }
              >
                Login
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  </RNCSafeAreaView>
</View>
`;
